cli/src/commands/list.ts

```diff
@@ -67,4 +73,6 @@ export const listFilesCommand = new Command('list')
         notFoundKeys
       )
     }
+
+    console.log(`\nIs immutable: ${isimmutable}`)
```

Comment:
IMO, we should create an issue to create a new command like "info" which shows some informations about an SC to offload this command which was supposed to just list files stored in the SC